core/whistle-1.0.0/src/wh_nodes.erl

```diff
@@ -415,10 +415,8 @@ handle_cast({'advertise', JObj}, #state{tab=Tab}=State) ->
 handle_cast({'gen_listener', {'is_consuming', 'true'}}
             ,#state{heartbeat_ref='undefined'}=State
            ) ->
-    Reference = erlang:make_ref(),
-    erlang:send_after(?HEARTBEAT, self(), {'heartbeat', Reference}),
     lager:debug("start heartbeat"),
-    {'noreply', State#state{heartbeat_ref=Reference, is_consuming='true'}};
+    handle_info({'heartbeat', 'undefined'}, State#state{is_consuming='true'});
```

Comment:
@k-anderson `wh_nodes` announces that it alive after timer with random timeout (5-15 seconds, as I can remember).
This change only for announcing that this node is alive (amqp_leader needs it) after start. It is a bit tricky, but I prefer to use existing code even usage is tricky.
`Amqp_leader` will work without this change, but it will fail to start several times (1-3) and will be restarted with supervisor.